### PR TITLE
Add disclaimer about multiple sway sessions

### PR DESCRIPTION
**Pull request by markstos**
I had started a second Sway session to debug why my first sway session was freezing, but then some other things broke.

Maybe this is so rare that it's not worth mentioning that it's not supported, but it seems worth documenting *somewhere*.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ This includes several areas of integration:
 
 - Managing sway environment. It's hard, opinionated and depends on the way user starts sway, so I don't have a solution that works for everyone and is acceptable for default configuration. See also [#6](https://github.com/alebastr/sway-systemd/issues/6).\
   The common solutions are `~/.profile` (if your display manager supports that), `~/.pam_environment`, or a wrapper script that sets the variables before starting sway.
+ 
+ - Supporting multiple concurrent sway sessions for the same user. It's uncommon and doing so would cause problems for which there are no easy solutions: To make this work, we set `WAYLAND_DISPLAY` with a `systemd --user` sesssion. This environment variable only accurate per-session, while `systemd --user` sessions are per-user. If the user starts a second Sway instance on the same machine, the new instance would overwrite this value, which could cause some services to break in the first sesssion if they were to read the variable after the second session starts.
```
Comment (Scrumplex) [inline README.md:19]: > To make this work, we set `WAYLAND_DISPLAY` with a `systemd --user` sesssion.

typo here ("sesssion")

Comment (Scrumplex) [inline README.md:19]: > `systemd --user` session

nit: maybe just call it "systemd user session"

Comment (Scrumplex) [inline README.md:19]: > If the user starts a second Sway instance on the same machine

nit: "on the same machine" doesn't sound right IMO

**Review by Scrumplex — CHANGES_REQUESTED** [review]
I don't think it's as uncommon as you think. You can also run Sway nested inside your currently running Sway session. That would also lead to the same issue.

**Comment by markstos** [comment]
You are welcome to take idea and modify the text as you see fit.  

Good comment about possibly nested sessions!

**Comment by markstos** [comment]
Maybe there's a way to check to see if the user attempts to use this tool with a second session and throw an error or print a warning if so? 

That would have helped me debug my problems today. I guess it was useful that I could still run a second Sway session, and I didn't really need any background systemd services to work there. So the behavior I would have liked would have been for the tool to notice that a Sway session had already been set up with systemd, and then just print a warning and not further modify the environment.

**Comment by Scrumplex** [comment]
It could be as simple as checking if `WAYLAND_DISPLAY" is present in the output of `systemctl --user show-environment`. This would also keep us from overriding it if other WMs or DEs set the variable (For example if the user is running Sway nested inside some other wayland compositor).

**Comment by alebastr** [comment]
> It could be as simple as checking if `WAYLAND_DISPLAY` is present in the output of `systemctl --user show-environment`. This would also keep us from overriding it if other WMs or DEs set the variable (For example if the user is running Sway nested inside some other wayland compositor).

`WAYLAND_DISPLAY` or `DISPLAY`. Sway can be nested inside of an X session and the same problem would appear.
One problem is that it requires making `--with-cleanup` behavior default. I was apprehensive of that as waiting for `swaymsg -t subscribe '["shutdown"]'` looks fragile, but enough time has passed and nobody reported any issues caused by this.

Overall I agree both with the disclaimer and with the script change. This week is quite busy though, so I don't know when I'll have enough time to implement and test it :disappointed:

**Comment by markstos** [comment]
Another option is to set an additional variable, like `SWAY_SYSTEMD_ACTIVE=1`. This would be cleaned up when sway exists. 

If that variable exists, warn and exit.

**Pull request merged**